WORKSPACE

```diff
@@ -110,22 +104,50 @@ load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
 
 sass_repositories()
 
+# Needed by gRPC.
+http_archive(
+    name = "build_bazel_rules_swift",
+    sha256 = "2245d21ba740f2b877d28fe97f77dbc7622942e09a7593ed748ce90afd95afd3",
+    strip_prefix = "rules_swift-0.25.0",
+    urls = [
+        "https://github.com/bazelbuild/rules_swift/archive/0.25.0.tar.gz",
+    ],
+)
+
+# gRPC.
+http_archive(
+    name = "com_github_grpc_grpc",
+    sha256 = "0f0b3a74af65049fbc6ef3072415ba96d73a7417bfa641fb85f43af9a158f57c",
+    strip_prefix = "grpc-2d8546a3c4ed8fa0a801770e58fd284cf4a70d7a",
+    urls = [
+        "https://github.com/grpc/grpc/archive/2d8546a3c4ed8fa0a801770e58fd284cf4a70d7a.tar.gz",
+    ],
+)
+
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
```

Comment:
I don't think I have `python{1,2,3}-dev` or `python2` on my workstation, and my python version is 3.9.9. Will investigate the issue further.

Reply:
Hmm, that's a bit odd. My python is also 3.9.9.  I guess there must be some configuration difference?

Maybe we should check w/ one other person just in case, but if nobody else is seeing this error, then I'm fine just proceeding and not mentioning it in DEVELOPMENT.md since the error is at least pretty self-explanatory about what to do to fix it.

Reply:
I was able to reproduce the same error after installing Python2, related issue: https://github.com/grpc/grpc/issues/21963.